src/containers/ExportModal.tsx

```diff
@@ -113,7 +113,7 @@ class PNGExportForm extends Component<PNGExportFormatProps> {
         <br/>
         <Checkbox name='alphaPixel' label='Alpha pixel work-around for Twitter' />
         <Checkbox name='borders' label='Include borders' />
-        <NumberInput name='scale' label='Scale' />
+        <NumberInput name='scale' label='Scale' defaultValue={1} />
```

Comment:
Did you check that the defaultValue is actually needed?  What happens if you leave it out?

I didn't try the code, but I've tried to write most of the code to be "controlled components" (value is passed via props, no need for defaultValues) and usually defaultValue is used only with uncontrolled.  My thinking is that as "scale" defaults to "1", this value should propagate to NumberInput without using defaultValue.

But it's been a while since I last touched this code so I could be wrong.

Reply:
I've just tried. Seems to work fine without it, scale of 1 is used by default. Though I was thinking this from the user perspective when i added the default value. To me at least it seems more clear what that field does if there is something in the box when the modal shows up.

Let me know and i can remove that if you disagree. :)

(Attached images of both)

![export-yes](https://user-images.githubusercontent.com/3483361/74091724-467e9380-4ac3-11ea-954f-9c0df6ebd610.jpg)
![export-no](https://user-images.githubusercontent.com/3483361/74091725-48485700-4ac3-11ea-80cb-648598299fbb.jpg)